components/plan-build/bin/hab-plan-build.sh

```diff
@@ -2002,6 +2021,7 @@ do_default_after() {
 # **Internal** Write the `$pkg_prefix/MANIFEST`.
 _build_manifest() {
   build_line "Creating manifest"
+  # shellcheck disable=2154
   if [[ -z $pkg_upstream_url ]]; then
```

Comment:
Why wasn't this quoted?

Reply:
Globbing and word-splitting are disabled for unary arguments to `[[` (but not `[`). `shellcheck` knows the difference and will warn if you change that to `[`.